app/helpers/custom_form_error_helper.rb

```diff
@@ -0,0 +1,19 @@
+module CustomFormErrorHelper
+  def errors(object, attribute)
+    error_messages = object.try(:errors).try(:full_messages_for, attribute)
+
+    if error_messages.is_a?(Array) and error_messages.count > 0
```

Comment:
__[RuboCop]__ Style/NumericPredicate: Use `error_messages.count.positive?` instead of `error_messages.count > 0`. ([link](https://sider.review/gh/repos/47168038/pulls/585#issue-4232500))

<sub>posted by Sider</sub>